src/test/java/org/gridsuite/modification/server/modifications/OperatingStatusModificationLockoutLineTest.java

```diff
@@ -79,35 +80,53 @@ protected void assertAfterNetworkModificationDeletion() {
+        //test lockout line modification with non-fictional switch
+        getNetwork().getSubstation("s1").newVoltageLevel().setId("v1f").setFictitious(true)
+                .setTopologyKind(TopologyKind.NODE_BREAKER).setNominalV(380.).add();
+        getNetwork().getSubstation("s2").newVoltageLevel().setId("v3f").setFictitious(true)
+                .setTopologyKind(TopologyKind.NODE_BREAKER).setNominalV(380.).add();
+
+        Line line = getNetwork().newLine()
+                .setId("nonFictionalDisconnect")
+                .setVoltageLevel1("v1f")
+                .setVoltageLevel2("v3f")
+                .setNode1(0)
+                .setNode2(0)
+                .setX(12)
+                .setR(7)
+                .add();
+        assertNotNull(line);
+
+        line.getTerminal("v1f").disconnect(SwitchPredicates.IS_NONFICTIONAL);
+        line.getTerminal("v3f").disconnect(SwitchPredicates.IS_NONFICTIONAL);
+        assertNotNull(line);
+        modificationInfos.setEquipmentId("nonFictionalDisconnect");
+        modificationInfos.setAction(OperatingStatusModificationInfos.ActionType.LOCKOUT);
+        String modificationJson = mapper.writeValueAsString(modificationInfos);
+        assertNull(getNetwork().getLine("nonFictionalDisconnect").getExtension(OperatingStatus.class));
+
+        mockMvc.perform(post(getNetworkModificationUri()).content(modificationJson).contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk());
+        assertNotNull(getNetwork().getLine("nonFictionalDisconnect").getExtension(OperatingStatus.class));
+
+    }
+
     @Test
     public void testCreateWithErrors() throws Exception {
         // line not existing
         OperatingStatusModificationInfos modificationInfos = (OperatingStatusModificationInfos) buildModification();
         modificationInfos.setEquipmentId("notFound");
         String modificationJson = mapper.writeValueAsString(modificationInfos);
-        mockMvc.perform(post(getNetworkModificationUri()).content(modificationJson).contentType(MediaType.APPLICATION_JSON))
-            .andExpect(status().isOk());
-        assertNull(getNetwork().getLine("notFound"));
-        assertLogMessage(new NetworkModificationException(EQUIPMENT_NOT_FOUND, "notFound").getMessage(),
-                modificationInfos.getErrorType().name(), reportService);
-
-        // modification action empty
-        modificationInfos.setEquipmentId("line2");
-        modificationInfos.setAction(null);
-        modificationJson = mapper.writeValueAsString(modificationInfos);
-        mockMvc.perform(post(getNetworkModificationUri()).content(modificationJson).contentType(MediaType.APPLICATION_JSON))
-            .andExpectAll(
-                    status().isBadRequest(),
-                    content().string(new NetworkModificationException(OPERATING_ACTION_TYPE_EMPTY).getMessage())
-            );
```

Comment:
Put back this test?

Reply:
Add:
* one test with a fictitious switch => it should throw an exception
* one test with a non breaker switch

Reply:
done